### PR TITLE
feat: Allow arbitrary agent addresses to be set

### LINKED ISSUE #1248: [FEATURE] Allow exchange agent address to be set dynamically
## Is your feature request related to a problem? Please describe.

Currently, the ibetExchange agent address is designed to be obtained from an environment variable, but it is not necessary to fix it in the system settings, and it would be better to extract the data of the agent to be referenced at each API request.

## Describe the solution you'd like
Add the agent address to the API query parameter.
Add it to the following APIs.

- /DEX/Market/OrderBook/Membership
- /DEX/Market/OrderBook/Coupon

### PR DESCRIPTION
**Pull request by YoshihitoAso**
close #1248

### CODE REVIEW

File: ENV_LIST.md [hunk 1]
```diff
@@ -42,7 +42,6 @@ The list of environment variables that can be set for this system is as follows.
 ### On-chain Exchange (Only for utility tokens)
 | Variable Name                               | Required | Details                                                                  | Example                                    | Default | 
 |---------------------------------------------|----------|--------------------------------------------------------------------------|--------------------------------------------|---------|
-| AGENT_ADDRESS                               | True*    | Paying agent address (*Set only if you use IbetExchange)                 | 0x0000000000000000000000000000000000000000 | --      |
```
Comment (YoshihitoAso) [inline ENV_LIST.md:45]: Deleted this variable.

File: app/api/routers/dex_market.py [hunk 2]
```diff
@@ -166,144 +166,44 @@ def list_all_membership_order_book(
...
+        filter(Order.exchange_address == exchange_address).
+        filter(Order.token_address == token_address).
+        filter(Order.is_cancelled == False)  # 未キャンセル
+    )
+
+    if is_buy:  # 買注文
+        query = query.filter(Order.is_buy == False)
+    else:  # 売注文
+        query = query.filter(Order.is_buy == True)
+
```
Comment (YoshihitoAso) [inline app/api/routers/dex_market.py:199]: Refactored duplicated business logic.

File: app/model/schema/node_info.py [hunk 3]
```diff
@@ -51,7 +51,6 @@ class GetNodeInfoResponse(BaseModel):
     ibet_security_token_escrow_abi: Optional[object]
     e2e_messaging_address: Optional[str]
     e2e_messaging_abi: Optional[object]
-    agent_address: Optional[str]
```
Comment (YoshihitoAso) [inline app/model/schema/node_info.py:54]: Deleted response item.

File: app/api/routers/dex_market.py (hunk 2 above)
Comment (purplesmoke05) [reply app/api/routers/dex_market.py:199]: I think `exchange_agent_address` (newly added) in request_query should be used to filter `Order` data.

Comment (YoshihitoAso) [reply app/api/routers/dex_market.py]: Sorry. Thanks for pointing that out.
I have fixed the query and added a test case.

**Pull request merged**